examples/layout-manager-playground/src/redux/store.ts

```diff
@@ -7,13 +7,14 @@ function configureStore() {
   const reducers = {};
   const INIT_STATE = {};
   const isMinimizeEnabled = true;
-  return createStore(
```

Comment:
Do you think it's possible to keep `createStore` with only a store created as before, and have a special `createMultipleStore` or any other name that would imply that there is multiple layout manager with the object as return? Like this, this would avoid the breaking change (even if it's not a big deal)

Reply:
Sounds good to me. I will add a new method